apps/src/templates/AgeDialog.jsx

```diff
@@ -14,17 +14,40 @@ import queryString from 'query-string';
  * We want users seeing that dialog to have to input their age, so using
  * a different session storage key here.
  */
-const AGE_DIALOG_SESSION_KEY = 'ad_anon_over13';
-const SONG_FILTER_SESSION_KEY = 'song_filter_on';
+export const AGE_DIALOG_SESSION_KEY = 'ad_anon_over13';
+export const SONG_FILTER_SESSION_KEY = 'song_filter_on';
 
-export const ageDialogSelectedOver13 = () => {
+const ageDialogSelectedOver13 = () => {
   return sessionStorage.getItem(AGE_DIALOG_SESSION_KEY) === 'true';
 };
 
-export const songFilterOn = () => {
+const songFilterOn = () => {
   return sessionStorage.getItem(SONG_FILTER_SESSION_KEY) === 'true';
 };
 
+/**
+ * The filter defaults to on. If the user is over 13 (identified via account or anon dialog), filter turns off.
+ */
+export const getFilterStatus = (userType, under13) => {
```

Comment:
I don't have a strong feeling about this, but just wondering what you think about moving this function to `dance/songs.js`? I was tracing through code to understand how songs are filtered and see in `songs.js` the function`getFilteredSongKeys` that uses the return from `getFilterStatus`.

Reply:
No strong opinion either - I can move this to songs.js if that's clearer! Good call with getFilteredSongKeys, these are definitely related.

Reply:
Yeah - I think it may be easier to understand how the function is been used. Thanks!

Reply:
Moved this function to `songs.js` and updated tests - I was able to convert `songsTest` to TS as well!